src/Functions/CustomWeekTransforms.h

```diff
@@ -106,8 +94,6 @@ struct ToStartOfWeekImpl
     {
         return time_zone.toFirstDayNumOfWeek(ExtendedDayNum(d), week_mode);
     }
-
-    using FactorTransform = ZeroTransform;
```

Comment:
But function `toStartOfWeek` is always monotonic and it's better to keep this behaviour.

Reply:
The same about `toYearWeek`.